transmart-rest-api/grails-app/controllers/org/transmartproject/interceptors/QueryAuditInterceptor.groovy

```diff
@@ -0,0 +1,17 @@
+package org.transmartproject.interceptors
+
+import groovy.transform.CompileStatic
+
+@CompileStatic
+class QueryAuditInterceptor extends AuditInterceptor {
+
+    QueryAuditInterceptor() {
+        match(controller: ~/query/,
+                action: ~/observations|counts|countsPerConcept|countsPerStudy|countsPerStudyAndConcept|aggregatesPerConcept|crosstable|table|supportedFields/)
```

Comment:
You could instead blacklist calls that should not be logged. Imagine the case when you extend the query controller, do we then expect that this interceptor is updated as well?